src/styleguide/components/s-api-mock-test.vue

```diff
@@ -154,32 +160,40 @@
 
     computed: {
       /**
-       * Returns list of endpoints  (filtered by search).
+       * Returns the mapped list of endpoints.
        */
-      filteredEndpoints(): object[] | null {
-        const { search, handlers } = this;
+      mappedEndpoints(): IEndpoint[] | null {
+        const { handlers } = this;
 
         if (!handlers) {
           return null;
         }
 
-        const mappedHandlers = handlers.map(handler => handler.info);
+        return handlers.map(handler => handler.info) as IEndpoint[];
+      },
+
+      /**
+       * Returns the filtered list of endpoints.
+       */
+      filteredEndpoints(): IEndpoint[] | null {
```

Comment:
suggestion: I would also recommend to always return an array here. If `mappedEndpoints` returns already one, you don't need a filter fallback.